drugs/templates/TargetSelectionTool.html

```diff
@@ -229,6 +230,8 @@ <h4>Expression (HPA)</h4>
                         <th style="text-align: center;">Drugs</th>
                         <th style="text-align: center;">Agents</th>
                         <th style="text-align: center;">Ligands</th>
+                        <th style="text-align: center;border-left: 1px solid #cccccc;">Small molecule</th>
+                        <th style="text-align: center;">Biologics</th>
```

Comment:
It might be just me, but what does Biologics mean under the Ligand type context? Is it the number of assays for that receptor ligand pair?